### PR TITLE
Organism slider

### PR DESCRIPTION
**Pull request by azeem110201**

### CODE REVIEW

File: src/components/organisms/Slider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export interface SliderProps {
+  data: SliderDataProps[];
+  selected?: number;
+  onClick?: () => void;
+  getSliderId: (id: number) => void;
+}
+
+const responsive = {
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
```
Comment (Sanjana1508) [inline src/components/organisms/Slider/index.tsx:24]: Give breakpoints in theme file

File: src/components/organisms/Slider/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  backgroundColor: theme.palette.structural[50],
+  display: "flex",
+});
+
+const Slider = (props: SliderProps) => {
+  const [select, setSelect] = useState<number>(
+    props.selected ? props.selected : 0
+  );
+
+  const handleSelect = async (id: number) => {
```
Comment (Sanjana1508) [inline src/components/organisms/Slider/index.tsx:50]: why do we need async for this function? There are no asynchronous functions in this

File: src/components/organisms/Slider/index.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      responsive={responsive}
+      slidesToSlide={3}
+      data-testid="slider"
+    >
+      {props.data.map((slider: SliderDataProps, index) => {
+        if (index !== props.data.length - 1) {
+          return (
+            <div key={index} data-testid="icon-typo">
+              <IconTypography
+                selected={select === index ? true : false}
```
Comment (Sanjana1508) [inline src/components/organisms/Slider/index.tsx:68]:
```suggestion
                selected={select === index}
```

File: src/components/organisms/Slider/index.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+          );
+        }
+      })}
+      {props.data.map((slider: SliderDataProps, index) => {
+        if (index === props.data.length - 1) {
+          return (
+            <div key={index} data-testid="icon-typo">
+              <IconTypography
+                selected={select === index ? true : false}
```
Comment (Sanjana1508) [inline src/components/organisms/Slider/index.tsx:84]:
```suggestion
                selected={select === index}
```

**Review by Sanjana1508 — CHANGES_REQUESTED** [review]

File: src/components/organisms/Slider/index.tsx (hunk 1 above)
Comment (azeem110201) [reply src/components/organisms/Slider/index.tsx:24]: These breakpoints are not mui breakpoints. These breakpoints are provided by custome 3rd party library.

Comment (Sanjana1508) [reply src/components/organisms/Slider/index.tsx:23]: Better to put them in a separate file, so that they can be reused

**Review by Sanjana1508 — APPROVED** [review]

**Pull request merged**